.github/workflows/lint.yml

```diff
@@ -96,3 +96,17 @@ jobs:
 
       - name: Run license checks
         run: addlicense -check -ignore '.git/**' -ignore 'build/**' -ignore 'llvm*/**' .
+
+  shell-script-check:
+    if: "github.event_name == 'pull_request'"
+    runs-on: ${{ github.repository == 'openxla/stablehlo' && 'ubuntu-22.04-64core' ||  'ubuntu-22.04'  }}
```

Comment:
What's up with these `runs-on` ?

Reply:
Not sure why these are necessary, but I think they ensure we run on an Ubuntu 22.04 VM: https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions

I just copied this to mirror the other steps.

Reply:
These are `(google runners || GH ubuntu runners)`

Reply:
Oh -- How do I learn more about google runners?
Is that part of the XLA repo?

Reply:
I am going to resolve this so I can submit the change, but I am also curious about this. @GleasonK do you know more about how our runners are configured?